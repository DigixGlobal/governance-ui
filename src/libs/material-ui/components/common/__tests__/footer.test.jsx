import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from '../footer';

describe('<Footer />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <Footer />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
