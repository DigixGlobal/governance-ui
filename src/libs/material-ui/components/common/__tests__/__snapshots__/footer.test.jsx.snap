// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<Segment
  attached="top"
  className="footer"
  color="black"
  compact={true}
  inverted={true}
  secondary={true}
  size="small"
>
  <Container>
    <Grid
      columns="equal"
      verticalAlign="middle"
    >
      <GridColumn>
        <ButtonGroup
          color="grey"
          size="mini"
        >
          <Button
            as="a"
            href="https://reddit.com/r/digix"
            icon="reddit alien"
            rel="noopener noreferrer"
            target="_blank"
          />
          <Button
            as="a"
            href="https://github.com/spectrum/spectrum"
            icon="github"
            rel="noopener noreferrer"
            target="_blank"
          />
          <Button
            as="a"
            href="https://www.cryptocompare.com/api/"
            icon="line chart"
            rel="noopener noreferrer"
            target="_blank"
          />
        </ButtonGroup>
      </GridColumn>
      <GridColumn
        textAlign="right"
      >
        <SpectrumWatermark />
      </GridColumn>
    </Grid>
  </Container>
</Segment>
`;
